app/src/main/java/com/epmedu/animeal/tabs/TabActivity.kt

```diff
@@ -51,6 +72,36 @@ class TabActivity : ComponentActivity() {
         }
     }
 
+    @Composable
+    private fun BottomAppBarFab(
+        currentRoute: String?,
+        associatedScreen: NavigationScreen = NavigationScreen.Home,
+        onNavigate: (NavigationScreen.Route) -> Unit
+    ) {
+        val route = associatedScreen.route
+
+        FloatingActionButton(
+            modifier = Modifier.padding(top = 22.dp),
+            shape = CircleShape,
+            onClick = {
+                onNavigate(route)
+            },
+            backgroundColor = when (currentRoute) {
+                route.name -> MaterialTheme.colors.primary
+                else -> MaterialTheme.colors.surface
+            },
+            contentColor = when (currentRoute) {
+                route.name -> MaterialTheme.colors.surface
+                else -> MaterialTheme.colors.onSurface
+            }
```

Comment:
On the design the fab has another color of the shadow, but I guess you can't easily change it, can you?

Reply:
I tried change shadow color, but it looks weird. Will discuss this moment with designers